api/ordinals/src/api/schemas.ts

```diff
@@ -0,0 +1,563 @@
+import { SwaggerOptions } from '@fastify/swagger';
+import { SERVER_VERSION } from '@hirosystems/api-toolkit';
+import { Static, TSchema, Type } from '@sinclair/typebox';
+import { TypeCompiler } from '@sinclair/typebox/compiler';
+import { SAT_SUPPLY, SatoshiRarity } from './util/ordinal-satoshi';
+
+export const OpenApiSchemaOptions: SwaggerOptions = {
+  openapi: {
+    info: {
+      title: 'Ordinals API',
+      description: `
+The [Ordinals API](https://docs.hiro.so/ordinals-api) is a service that indexes Bitcoin Ordinals data and exposes it via REST API endpoints.
+
+Here are the key features of the Ordinals API:
+
+- **Ordinal Inscription Ingestion**: The Ordinals API helps with the complete ingestion of ordinal inscriptions. Using our endpoints, you can retrieve the metadata for a particular inscription, all inscriptions held by a particular address, trading activity for inscriptions, and more.
+
+- **BRC-20 Support**: The Ordinals API also offers support for BRC-20 tokens, a fungible token standard built on top of ordinal theory. Retrieve data for a particular BRC-20 token, a user's BRC-20 holdings, marketplace activity, and more.
+
+- **REST JSON Endpoints with ETag Caching**: The Ordinals API provides easy-to-use REST endpoints that return responses in JSON format. It also supports ETag caching, which allows you to cache responses based on inscriptions. This helps optimize performance and reduce unnecessary requests.
+
+
+The source code for this project is available in our [GitHub repository](https://github.com/hirosystems/ordinals-api). You can explore the codebase, contribute, and raise issues or pull requests.
```

Comment:
Should we link to the root of the monorepo?

```suggestion
The source code for this project is available in our [GitHub repository](https://github.com/hirosystems/ordhook). You can explore the codebase, contribute, and raise issues or pull requests.
```